### ISSUE TITLE
Add support for setting default refresh rate for Dashboards

### ISSUE THREAD

**Issue by gaby** [openobserve/openobserve#3780]
### Which OpenObserve functionalities are relevant/related to the feature request?

dashboards

### Description

Add support in the dashboard settings for setting the default Refresh Rate. Similar to how we can now add default date-time setting.

### Proposed solution

Similar to PR #3421 but a field for setting default Refresh Rate of the dashboard:

### Alternatives considered

Changing the refresh rate manually every time I open a dashboard.

**Comment by gaby** [comment]
@ktx-kirtan Any updates on this? Been 1.5 years with no updates.

It's a very bad user experience having to click Refresh or Query every time I open a dashboard.

**Comment by ktx-kirtan** [comment]
Yes, we will be picking this up for the next release. Not on the bugfix release, but the next feature release.

**Comment by gaby** [comment]
@ktx-kirtan Any news? on a daily basis I have to set the refresh interval around 50-80 times...

**Comment by gaby** [comment]
Was told via Slack that this requires a lot of changes. Just going to close it.

**Comment by ethanmontgomery148-hub** [comment]
Thanks for the update—understood. If it gets revisited or you’d like help
breaking down the changes, happy to take a look.
